Lombiq.Hosting.Tenants.Management/Views/ShellSettingsEditor.cshtml

```diff
@@ -1,15 +1,20 @@
-﻿@using Newtonsoft.Json
+﻿@using Lombiq.Hosting.Tenants.Management.Controllers
+@using Newtonsoft.Json
 @using OrchardCore
+@using OrchardCore.Mvc.Core.Utilities
+@using static Lombiq.Hosting.Tenants.Management.Constants.FeatureNames
+
 @model Lombiq.Hosting.Tenants.Management.Models.ShellSettingsEditorViewModel
 
-<form asp-action="Edit"
-      asp-controller="ShellSettingsEditor"
-      asp-area="Lombiq.Hosting.Tenants.Management"
+<form asp-action="@nameof(ShellSettingsEditorController.Edit)"
+      asp-controller="@typeof(ShellSettingsEditorController).ControllerName()"
+      asp-area="@Module"
       method="post"
       class="no-multisubmit">
     <div class="@Orchard.GetWrapperCssClasses("field-wrapper")" id="FieldWrapper">
         <label asp-for="Json" class="@Orchard.GetLabelCssClasses()">@T["Shell Settings editor"]</label>
         <div class="@Orchard.GetEndCssClasses()">
+            <span class="hint">@T["Key-value pairs should be set here. Save empty string value to delete setting."]</span>
```

Comment:
Removed the second sentence due to https://github.com/Lombiq/Hosting-Tenants/pull/84#discussion_r1346223357.

```suggestion
            <span class="hint">
                @T["JSON key-value pairs should be set here, in the same way you'd configure the tenant-specific section in an appsettings.json file."]
            </span>
```

Reply:
This is not entirely true, because in the appsettings.json file you create nodes for the separate levels, but here you add key value pairs on the same level.
So here you need to use this format e.g:
```
{
  "OrchardCore_Media_Azure:ConnectionString": "UseDevelopmentStorage=true",
  "OrchardCore_Media_Azure:ContainerName":  "osocemedia"
}
```

Reply:
No, currently you can't. If you want this, let me know. I went with the easier solution because it wasn't specified otherwise.

I need to readd this parser [https://github.com/Lombiq/Hosting-Tenants/pull/84/commits/d7da847459e9d00154fef30143686f352bf3ca83](https://github.com/Lombiq/Hosting-Tenants/pull/84/commits/d7da847459e9d00154fef30143686f352bf3ca83)
Then I need to change the logic in the editor filter, and in the controller.